### DISCUSSION TITLE
Questions related to the assestment

### DISCUSSION THREAD

**Discussion by JUNEDSK** [TDRA-ae/aegov-dls#61] [Q&A] [answered]
Hello there,

I have listed some points; could you please explain them? Any references provided would be appreciated.

- Testing sections of the website's background and foreground color usage to be 3:1?
- Testing contrast ratio of action elements background and foreground to be 4.5:1?
- The design process ensured spacing between sections as requited by WCAG 2.1 and mentioned in the guideline?

**Comment by moiz-conceptlz** [comment, marked as answer]
Hello @JUNEDSK 

The contrast ratio is part of WCAG's Success Criterion - specifically SC 1.4.3. You can read in detail about this on https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html

Similarly, Spacing is part of SC 1.4.12 - which you can refer to here - https://www.w3.org/WAI/WCAG22/Understanding/text-spacing.html

I hope this was helpful
